app/assets/javascripts/listings/ListingIdentityService.js.coffee

```diff
@@ -19,6 +19,10 @@ ListingIdentityService = (ListingConstantsService) ->
     return false unless listing
     listing.Tenure == 'New sale' || listing.Tenure == 'Resale'
 
+  Service.isRental = (listing) ->
```

Comment:
good idea! do you think we should be moving in the direction of using .isRental and .isSale instead of just .isSale and !.isSale?